backend/de.metas.contracts/src/main/java/de/metas/contracts/FlatrateTermPricing.java

```diff
@@ -84,20 +85,31 @@ public IPricingResult computeOrThrowEx()
 
 	private PriceListId retrievePriceListForTerm()
 	{
+		final IBPartnerDAO bpartnerDAO = Services.get(IBPartnerDAO.class);
```

Comment:
the simplest "fix" is to have the services as parameter. 
scrolling down i saw u have more services...

Reply:
As DM-ed, I will not pass the services as parameters here in this issue because there aer too many places where I should change. 
Moved the old service at the beginning of the method, though.